src/django_upgrade/fixers/admin_register.py

```diff
@@ -36,10 +36,11 @@
 
 
 class AdminDetails:
-    __slots__ = ("parent", "model_names_per_site")
+    __slots__ = ("parent", "node", "model_names_per_site")
 
-    def __init__(self, parent: ast.AST) -> None:
+    def __init__(self, parent: ast.AST, node: ast.ClassDef) -> None:
         self.parent = parent
+        self.node = node
```

Comment:
Could maybe only track `node.lineno` ? It seems like the only thing you actually use later?